horace_core/sqw/@rundatah/private/rundata_from_sqw_.m

```diff
@@ -44,31 +44,34 @@
 
 
 lattice = oriented_lattice();
-lattice.alatt = header.samples{1}.alatt;
-lattice.angdeg = header.samples{1}.angdeg;
-lattice.u      = header.expdata(1).cu;
-lattice.v      = header.expdata(1).cv;
-lattice.psi    = header.expdata(1).psi*(180/pi);
-lattice.omega = header.expdata(1).omega*(180/pi);
-lattice.dpsi  = header.expdata(1).dpsi*(180/pi);
-lattice.gl    = header.expdata(1).gl*(180/pi);
-lattice.gs    = header.expdata(1).gs*(180/pi);
+lattice.alatt = exp_inf.samples{1}.alatt;
+lattice.angdeg = exp_inf.samples{1}.angdeg;
```

Comment:
this line and subsequent assumes only one run in the exp_inf. This may be true for the particular function here but it needs commenting so the reviewer is not wondering what is happening to the other runs

Reply:
this function currently works for single file sqw only. the extension to multiple files is non-trivial and I am not sure if is necessary.